src/background.js

```diff
@@ -243,6 +243,13 @@ chrome.alarms.onAlarm.addListener(function(alarm) {
     eventIndex = index;
     return event.event_id === alarm.name;
   });
+  var reminderMinutes = feeds.events[eventIndex].reminders[0].minutes;
```

Comment:
Here and below: we only process `reminders[0]`. Shouldn’t this iterate over all `reminders` for the event?

Reply:
Yes, we should iterate over all reminders. I will change it to handle this